api/v1alpha4/common_types.go

```diff
@@ -79,6 +79,14 @@ const (
 
 	// InterruptibleLabel is the label used to mark the nodes that run on interruptible instances
 	InterruptibleLabel = "cluster.x-k8s.io/interruptible"
+
+	// ExternallyManagedAnnotation is an annotation that can be applied to InfraCluster resources to signify that
```

Comment:
```suggestion
	// ExternallyManagedAnnotation is an annotation that can be applied to InfraCluster CRs to signify that
```

Reply:
This will clarify that this is referring to InfraCluster CRs and not cloud infra resources

Reply:
Do we need to use an acronym here? `resources` is clear imo

Reply:
I also think this is clearer as `resources`, gonna leave this one as is